python/oauth-demo/README.md

```diff
@@ -0,0 +1,85 @@
+# OAuth Demo
+
+The OAuth Demo is an application that demonstrates authorization with a user's Asana account via a basic OAuth server (built with [Flask](https://flask.palletsprojects.com/en/stable/)). The application shows how you might send a user through the [user authorization endpoint](https://developers.asana.com/docs/oauth#user-authorization-endpoint), as well as how a `code` can be exchanged for a token via the [token exchange endpoint](https://developers.asana.com/docs/oauth#token-exchange-endpoint).
+
+_Note: This OAuth server should only be used for testing and learning purposes._
+
+Documentation: Asana's [OAuth](https://developers.asana.com/docs/oauth)
+
+## Requirements
+The application was built with Python 3.9.0.
+
+Visit Python's [official website](https://www.python.org/) to get the latest version for your local machine.
+
+### Dependencies:
+- [Flask](https://flask.palletsprojects.com/en/stable/)
+- [Requests](https://pypi.org/project/requests/)
+- [python-dotenv](https://pypi.org/project/python-dotenv/)
+- [Asana Python library](https://pypi.org/project/asana/)
+
+## Installation
+
+After cloning this project, navigate to the root directory and install dependencies:
+
+```
+pip install -r requirements.txt
+```
+
+## Usage
+
+1. [Create an application](https://developers.asana.com/docs/oauth#register-an-application). Take note of your **client ID** and **client secret**, and set the **redirect URI** to `http://localhost:3000/oauth-callback`. 
+
+_Note: In order for a user to be able to authorize via the [user authorization endpoint](https://developers.asana.com/docs/oauth#user-authorization-endpoint), the application must be available in the user's workspace. See the [manage distribution](https://developers.asana.com/docs/manage-distribution) documentation for details._ 
+
+2. Create a `./.env` file (in the root directory of the project) with the required configurations:
```

Comment:
maybe just call it `.env` because you already mentioned `(in the root directory of the project)`